packages/laconia-event/src/apigateway/parseWebsocket.js

```diff
@@ -0,0 +1,7 @@
+const getBody = require("./getBody");
```

Comment:
File name is still with small letter s i.e. supposed to be parseWebSocket. There are other files with this issue too.